### DISCUSSION TITLE
Request: Guidance on Playtime Synchronization

### DISCUSSION THREAD

**Discussion by sino1641** [neon-nyan/Collapse#600] [Q&A] [answered]
Hi,
Thanks for the great update. 
I have questions on the new feature: Guidance on Playtime Synchronization cross devices.
1. Can I use the official db-collapse.turso.io or should I register one by myself?
    - Token cannot be empty so I assume this is not possible.
1. I am trying it on my own turso account, but it seems I cannot sync the data.
    1. A testkey will be pop up when testing the db connection.
       <img width="674" alt="Snipaste_2024-11-03_20-02-14" src="https://github.com/user-attachments/assets/92d134a6-9f84-4236-a692-4e9052330403">

    1. Nothing updated after I clicked the sync button manually but only got two log:
    [Info]    [CollapsePlaytime::DbSync] Starting sync operation...
    [Info]    [DbHandler::Init] Reinitializing database system...

Therefore, it would be more than appreciated if you could provide guidance on this new feature with more details.

**Comment by bagusnl** [comment, marked as answer]
1. For privacy reason, we have decided to not share our own instance of the database. So you should register one yourself
2. If the test key is there and Collapse says that the connection is okay, you should be fine. Try launching a game and check the database table again. See if new fields appears
